web-animations-2/Overview.bs

```diff
@@ -2448,16 +2448,17 @@ partial dictionary ComputedEffectTiming {
     CSSNumberish         endTime;
     CSSNumberish         activeDuration;
     CSSNumberish?        localTime;
-    CSSNumberish?        progress;
 };
 </pre>
 
 <div class='members'>
 
 :   <dfn dict-member for=ComputedEffectTiming>startTime</dfn>
 ::  The <a lt='animation effect start time'>start time</a> of this
-    <a>animation effect</a> in milliseconds or as a percentage of the
+    <a>animation effect</a> expressed as a double in milliseconds
+    if [=timeline duration=] is unresolved or as a percentage of
```

Comment:
We seem to be using an unresolved timeline duration as an alternative to associated with a progress based timeline, however the latter is a better reason for the startTime to be expressed as a percentage - whereas if we had a finite time-based timeline I could see the time being expressed either way and most likely using a double for compatibility.

Reply:
Switched to being explicit about using % only with a progress based timeline.  Switched ordering to avoid negative condition or otherwise, which feels awkward.